src/app/page/page.component.ts

```diff
@@ -1,5 +1,8 @@
 namespace Page {
-
+  enum state {
```

Comment:
Lets capitalise a enum name. Also, `state` doesn't make sense in the context of Page namespace. Use a more specific name, such as `VerticalNavState`.

In programming, always be careful when you are naming something. It's very important to be aware of the **scope** of the name.